lib/serial_terminal.pm

```diff
@@ -232,4 +236,37 @@ sub upload_file {
       or die("Failed to finally copy file from '$tmpfilename' to '$dst'");
 }
 
+
+sub reboot {
+    my (%args) = @_;
+    $args{check} //= 1;
```

Comment:
```suggestion
    $args{record_uptime} //= 1;
```

Reply:
ok I haven't read till the end , so ignore this suggestion :)

Reply:
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13930#discussion_r775924340 it is really a validation, that the reboot took place and not just a `CTRL+D` happen.